docker-compose/Readme.md

```
@@ -70,17 +72,48 @@ Let's look at the "MOSIP ID" example to understand how these properties work tog
+
+`isAuthRequestEmbedded: true`: The corresponding VP request will use `client_id_scheme` as `pre_registered` and Auth Request will be available to wallet directly within the VP request.
+
 `definition` : The presentation definition for the particular type of credential. For more details check [[DIF.PresentationExchange]](https://identity.foundation/presentation-exchange/spec/v2.0.0/)
 
-### Run Using Docker Compose:
+---
+
+### OpenID4VP Setting Up Proxy For Localhost
+
+To get the OpenID4VP flow working locally, use a proxy service like ngrok or localtunnel 
+to create a proxy url like https://proxyurl.ngrok.app for http://localhost:3000.
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)